src/clouds.ts

```diff
@@ -1,3 +1,9 @@
+import { AzureSettings, config } from '@grafana/runtime';
+
+interface AzureSettings2 extends AzureSettings {
```

Comment:
I'd leave a TODO comment to ensure this type is removed in the future

Reply:
Added TODO and sent PR to `@grafana/runtime`.